src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -266,6 +280,95 @@ def __init__(self, session: "snowflake.snowpark.session.Session") -> None:
         ] = None
         self._infer_schema_target_columns: Optional[List[str]] = None
 
+    def load(
+        self,
+        path: str,
+        format: Optional[
+            Literal["avro", "csv", "json", "xml", "parquet", "orc"]
+        ] = None,
+        schema: StructType = None,
+        **kwargs,
+    ) -> DataFrame:
+        """
+        Loads data from a file path.
+
+        This method is typically the last method in a daisy chain of DataFrame reader methods.
+        It allows specifying the file type using the `format` parameter, providing a schema using the `schema` parameter,
+        and passing any copy or format-specific options using the `kwargs` parameter.
```

Comment:
```suggestion
        This method is typically the last method in a daisy chain of ``DataFrame`` reader methods.
        It allows specifying the file type using the ``format`` parameter, providing a schema using the ``schema`` parameter,
        and passing any copy or format-specific options using the ``kwargs`` parameter.
```

Reply:
Thanks. I learnt a lot about docstring formatting. I hope it is right now.